tools/commissioner_thci/commissioner_impl.py

```diff
@@ -401,6 +404,34 @@ def getCOM_TOK(self):
         result = self._execute_and_check('token print')
         return self._hex_to_bytes(result[0])
 
+    def getCommissioningLogs(self):
+        processed_logs = []
+        thci_logs = self._command("grep \"\\[ thci \\]\" {}".format(
+            self.log_file))
```

Comment:
how do you clean `log_file` if we are running tests one by one?

Reply:
The log file is overridden for each run of the commissioner.